docs/usage/self-hosted-configuration.md

```diff
@@ -487,3 +487,13 @@ This is currently applicable to `npm` and `lerna`/`npm` only, and only used in c
 ## username
 
 Mandatory if a GitHub app token is in use using the CLI.
+
+## writeDiscoveredRepos
+
+Optional parameter which allows to write the discovered repositories into a JSON file instead of renovating them.
+
+Usage: `renovate **--write-discovered-repos**=/tmp/renovate-repos.json`
```

Comment:
Doesn't it look like this in markdown?

![image](https://user-images.githubusercontent.com/6311784/134290196-77ff7d2e-e0cc-4579-b9b4-7f1d534ba6df.png)

Reply:
:thinking:  prettier keeps adding those. I have assumed that this is a syntax I'm not aware of.

Reply:
prettier?? does it think you are trying to do "emphasis" with `--` and replaces it?